.github/workflows/build-test.yml

```diff
@@ -0,0 +1,22 @@
+name: build 
+
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.21'
+    - name: image
+      uses: redhat-actions/buildah-build@v2
+      id: build-image
+      with:
+        image: splat-bot
+        tags: latest ${{ github.sha }}
+        containerfiles: |
+          ./Containerfile      
```

Comment:
need to move from Dockerfile to Containerfile
and that was just a placeholder copy-pasta, so needs to be changed